recipes/gdal/post_3.5.0/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+cmake_minimum_required(VERSION 3.15)
+project(gdal_cmake_wrapper)
+
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR})
+include(conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+include(CMakePushCheckState)
+
+
+if (${GDAL_USE_POPPLER})
+    find_package(poppler)
+    set(Poppler_VERSION_STRING ${poppler_VERSION})
+    add_library(Poppler::Poppler ALIAS CONAN_PKG::poppler)
```

Comment:
`CONAN_PKG` is deprecated and we should avoid it, use correct project target 🙏

Reply:
I did not know it was deprecated. I started removing those.